Add Table Store 0.2.0 release (#562)
apache/flink-web: _posts/2022-08-08-release-table-store-0.2.0.md

---
layout: post
title:  "Apache Flink Table Store 0.2.0 Release Announcement"
subtitle: "For building dynamic tables for both stream and batch processing in Flink, supporting high speed data ingestion and real-time data query."

> date: 2022-08-08T08:00:00.000Z

Reviewer: Minor: please remember to update the date accordingly

categories: news
authors:
- JingsongLi:
  name: "Jingsong Lee"
---

<!-- improve style of tables -->
<style>
  table { border: 0px solid black; table-layout: auto; width: 800px; }
  th, td { border: 1px solid black; padding: 5px; padding-left: 10px; padding-right: 10px; }
  th { text-align: center }
  td { vertical-align: top }
</style>

The Apache Flink community is pleased to announce the release of the
[Apache Flink Table Store](https://github.com/apache/flink-table-store) (0.2.0).

Please check out the full [documentation]({{site.DOCS_BASE_URL}}flink-table-store-docs-release-0.2/) for detailed information and user guides.

<br/>

## What is Flink Table Store

> Flink Table Store is a data lake storage for streaming updates/deletes changelog ingestion and high-performance queries in real time.

Reviewer: It seems the description in this blog post is different from the readme file of the [master branch](https://github.com/apache/flink-table-store/blob/master/README.md) and to-be-released source package, and suggest keeping them in sync.

```sql
CREATE CATALOG tablestore WITH (
  'type'='table-store',
  'warehouse'='hdfs://nn:8020/warehouse/path',
  -- optional hive metastore
  'metastore'='hive',
  'uri'='thrift://<hive-metastore-host-name>:<port>'
);

USE CATALOG tablestore;

CREATE TABLE my_table ...
```

### Ecosystem

In this release, we provide support for Flink 1.14 and provide read support for multiple compute engines.

 Engine    | Version  | Feature     |    Read Pushdown   |
---------- | -------- | ----------- | ------------------ |
 Flink     | 1.14     | read, write | Projection, Filter |
 Flink     | 1.15     | read, write | Projection, Filter |
 Hive      | 2.3      | read        | Projection, Filter |
 Spark     | 2.4      | read        | Projection, Filter |
 Spark     | 3.0      | read        | Projection, Filter |
 Spark     | 3.1      | read        | Projection, Filter |
 Spark     | 3.2      | read        | Projection, Filter |
 Spark     | 3.3      | read        | Projection, Filter |
 Trino     | 358      | read        | Projection, Filter |
 Trino     | 388      | read        | Projection, Filter |

<br/>

### Append-only

> Append-only tables are a performance feature that only accepts INSERT_ONLY data to append to the storage

Reviewer: ```suggestion
The append-only table feature is a performance improvement and only accepts INSERT_ONLY data to append to the storage
```
